lib/multitask.py

```diff
@@ -115,11 +112,12 @@ def get_losses(datasets, num_train_ds, label_smoothing=0):
   return losses, loss_weights
 
 def get_losses_and_callbacks(
-    datasets, num_tr_datasets, prefix, train_params, add_mw_callback=False,
-    end_lr_coefficient=0.1):
-  lr_schedule = training.get_lr_schedule(
-      train_params.lr, train_params.num_epochs,
-      end_lr_coefficient=end_lr_coefficient)
+    datasets, num_tr_datasets, prefix, train_params, add_mw_callback=False):
+
+  lr_schedule = tf.keras.optimizers.schedules.ExponentialDecay(
+      train_params.lr,
+      decay_steps=train_params.num_steps,
+      decay_rate=0.99)
```

Comment:
This should ideally be a flag.

Reply:
Do you mean that there should be a way to disable the exponential decay?